### ISSUE TITLE
Properly assign/unassign upload tag(s) on incremental uploads

### ISSUE THREAD

**Issue by mwinokan** [m2ms/fragalysis-frontend#1517]
@kaliif I am not sure what the current implementation of the "New" tag that is added to all LHS observations by default, but it should be the following please:

On every upload, please tag all observations with a tag corresponding to their version i.e. create tags `upload_1`, `upload_2`

**Comment by phraenquex** [comment]
@kaliif also need tags that reflect the date of the upload.

**Comment by mwinokan** [comment]
@kaliif do you already track which uploads correspond to which datasets in the database? 

If so, I think we keep this ticket to just assigning the automatic 'upload_*' tags. Else, I think the loader and schema changes should go into #1524

**Comment by kaliif** [comment]
@mwinokan yes, datasets are tracked and the backend can tell which upload it came from.

@phraenquex does the tag need to contain the date? How should it be formatted?

**Comment by phraenquex** [comment]
@mwinokan if it's a quick win to add the date, then I'd say: 

`upload_* yyyy-mm-dd`

Then at least text search should work.

**Comment by mwinokan** [comment]
@kaliif says it's a quick one-line fix to add the date, @phraenquex asks for this to be implemented before handling a bigger F/E change

**Comment by mwinokan** [comment]
@kaliif has implemented the `upload_?` spec in staging and it looks ok. Date appending is still pending

**Comment by phraenquex** [comment]
@mwinokan if "date appending" is "still pending", then please move this back to "In progress", and @kaliif can you please do it as first priority - so that it can go into production.

(Unless @mwinokan overrides me and says that the datestamp is not necessary - but I didn't hear anybody object before, and Kalev sayid it's a very easy change, as far as I remember.)

**Comment by mwinokan** [comment]
@kaliif needs to check if this was implemented and then deploy it to staging

**Comment by mwinokan** [comment]
Confirmed implemented in staging
